core/views.py

```diff
@@ -47,6 +55,7 @@ def viewNewsletters(request):
     })
 
 
+<<<<<<< HEAD
```

Comment:
`AccountTabsMixin` actually no longer exists with the new setup. You can move your `AccountChangeView` to `user_interaction/account_pages/views.py` and have your `AccountChangeView` inherit from `AccountViewMixin` instead as well (see how this is done for `SiteAccountView` for instance)